### ISSUE TITLE
(Self-Voicing) Content still accessible to hovers and clicks when Options dialog menu open

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#224]
Hover of content underneath shows through Options Menu
<img width="976" alt="Screen Shot 2020-08-06 at 5 41 20 PM" src="https://user-images.githubusercontent.com/1902116/89578063-2dedfc80-d80c-11ea-9a13-c5938ff9ad45.png">
<img width="968" alt="Screen Shot 2020-08-06 at 5 41 58 PM" src="https://user-images.githubusercontent.com/1902116/89578070-2fb7c000-d80c-11ea-99b5-97f950edb82a.png">

And when the close button was pressed to close the Options Menu, the dialog closed and the Force reading block self-voiced at the same time (no actual highlighting happened, but it self-voiced).
<img width="976" alt="Screen Shot 2020-08-06 at 5 41 20 PM" src="https://user-images.githubusercontent.com/1902116/89578359-9341ed80-d80c-11ea-8788-bc0287879c08.png">

Content is being self-voiced in the video when this screen shot is taken right after participant clicks the close button.
<img width="868" alt="Screen Shot 2020-08-06 at 5 46 02 PM" src="https://user-images.githubusercontent.com/1902116/89578459-bd93ab00-d80c-11ea-9fc7-607b27f238da.png">

**Comment by jessegreenberg** [comment]
@terracoda I am afraid this one could be a bit tricky to resolve with the current implementation. Can you comment on how important this is to resolve for this prototype? If not critical it may be best to come back to this when we are ready to move beyond prototype phase and invest in a more robust solution.

**Comment by terracoda** [comment]
It's critical for publication, but not critical for prototype.

**Comment by jessegreenberg** [comment]
OK thanks - I am going to put this on hold relative to the other self-voicing issues since it will likely take a substantial amount of time.

**Comment by jessegreenberg** [comment]
I came back to this again after #221 because it is getting much more annoying. It is apparent that we need to leverage scenery hit detection more for this feature. But there are many things that need different mouse/touch areas for self-voicing content than for interaction 'normal' user input. And we want hits to be detected on areas that were previously not hittable. If we have to use Nodes for hit detection for cases like this, will we end up adding `Rectangle`s around all the non-interactive things that need a hit test? I liked ShapeHitDetector it made that unecessary.

I also considered adding this code, but it is too hacky and I don't want to reach into private modalNodeStack of Sim.js. It also certainly wouldn't work if we ever want content in the dialog to have self voicing content itself.

```js
    if ( phet.joist.sim.modalNodeStack.length > 0 ) {
      this.property.set( false );
    }
```

**Comment by terracoda** [comment]
The highlights are no longer showing through @jessegreenberg. Can we close this issue?

**Comment by jessegreenberg** [comment]
Yes, thanks - this issue has been solved.
